recirq/qml_lfe/dynamics_fuzz_test.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pytest
+import tempfile
+import numpy as np
+
+
+def test_shadows_dont_seperate(tmpdir):
```

Comment:
This seems like a fairly in-depth test.  Might it be worth a short docstring to explain what it is doing?

Reply:
Added a brief explanation.